spread.yaml

```diff
@@ -46,27 +46,25 @@ backends:
         systems:
             - ubuntu-14.04-64:
                 kernel: GRUB 2
-                workers: 4
+                workers: 5
```

Comment:
I don't think it is needed, the error in https://travis-ci.org/snapcore/snapd/builds/252773594?utm_source=github_status&utm_medium=notification it is because linode machines went down and discarded, it is another problem that we have to fix on spread  or linode.
Same for all the workers